File: csv/rhino/read/csv_read.js
// ------------------------------------------------------------------
//
//	csv_read.js
//
//						Feb/21/2011
//
// ------------------------------------------------------------------
importPackage (java.io);

load ("/var/www/data_base/common/rhino_common/file_io.js");
load ("/var/www/data_base/common/rhino_common/text_manipulate.js");

file_in=arguments[0];

print	("*** 開始 ***");

var dict_aa = csv_read_proc (file_in);

dict_display_proc (dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: csv/rep/read/csv_read.jl
;; --------------------------------------------------------------------
;;	csv_read.jl
;;
;;						Mar/02/2011
;;
;; --------------------------------------------------------------------
(require 'tables)
(load-file "/var/www/data_base/common/rep_common/text_manipulate.jl")
;; --------------------------------------------------------------------
(format t "*** 開始 ***")
(setq file_in (cadr command-line-args))
(format t file_in)
(setq dict_aa (make-table string-hash string=))
(let ((port_in (open-file file_in 'read)))
	(csv_read_proc port_in dict_aa)
)

(table-walk dict_display_proc_single dict_aa)

(format t "*** 終了 ***")

;; --------------------------------------------------------------------
